packages/devtools/test/integration_tests/server_test.dart

```diff
@@ -12,10 +13,13 @@ import '../support/cli_test_driver.dart';
 import '../support/devtools_server_driver.dart';
 import 'integration.dart';
 
-void main() {
-  CliAppFixture appFixture;
-  DevToolsServerDriver server;
+CliAppFixture appFixture;
+DevToolsServerDriver server;
+final Map<String, Completer<Map<String, dynamic>>> completers =
```

Comment:
remove the massive generic types from the RHS.

Reply:
> 2. VS Code currently passed `--port=0` to get a random port, to avoid issues with the port already being in use. Chrome saves notification permissions based on host/port which means the user will be prompted to approve notifications each time it spawns on a port they haven't accepted before. Switching to a hard-coded port would avoid this, but may introduce other issues.

We should switch to a hard coded port with devtools falling back to increasing port #s. We've been too aggressive about using a random port #. The other option long term is to actually serve devtools from a website which would fix the issue.

Reply:
I've opened https://github.com/flutter/devtools/issues/1037 to try ascending ports - and I'll work on that and land it before this one lands.